demo-java/app/src/main/java/com/example/placesdemo/programmatic_predictions/ProgrammaticAutocompleteToolbarActivity.java

```diff
@@ -114,20 +147,32 @@ private void initRecyclerView() {
         recyclerView.setAdapter(adapter);
         recyclerView
             .addItemDecoration(new DividerItemDecoration(this, layoutManager.getOrientation()));
+        adapter.setPlaceClickListener(this::geocodePlaceAndDisplay);
     }
 
     /**
-     * This method demonstrates the programmatic approach to getting place predictions
+     * This method demonstrates the programmatic approach to getting place predictions. The
+     * parameters in this request are currently biased to Kolkata, India.
      *
-     * @param query the plus code query string (e.g. "MC2W+RG K")
+     * @param query the plus code query string (e.g. "GCG2+3M K")
      */
     private void getPlacePredictions(String query) {
+
+        // The value of 'bias' biases prediction results to the rectangular region provided
+        // (currently Kolkata). Modify these values to get results for another area.
```

Comment:
Also mention .setCountries as another critical parameter for delivering good/expected results.